gitops/argocd/tekton-results/statefulset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/part-of: tekton-results
+    app.kubernetes.io/version: v0.3.0
+  name: postgres
+  namespace: openshift-pipelines
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgres
+      app.kubernetes.io/part-of: tekton-results
+      app.kubernetes.io/version: v0.3.0
+  serviceName: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+        app.kubernetes.io/part-of: tekton-results
+        app.kubernetes.io/version: v0.3.0
+    spec:
+      containers:
+        - envFrom:
+            - configMapRef:
+                name: tekton-results-postgres
+            - secretRef:
+                name: tekton-results-postgres
+          image: postgres@sha256:6647385dd9ae11aa2216bf55c54d126b0a85637b3cf4039ef24e3234113588e3
```

Comment:
Could we use a more user friendly tag, such as a version number?

Reply:
This image name is directly taken from the last stable release version on Tekton Results. It would be preferable to keep that since there have been some problems with incompatible Postgres images.